### PR TITLE
Remove two way binding for composition with respect of backward compatibility

### PR DESCRIPTION
**Pull request by PatrykSzwer**
Needed for https://github.com/Starcounter/Blending/pull/395

### CODE REVIEW

**Review by warpech — CHANGES_REQUESTED** [review]
Pls make the code change in `updateComposition` DRY. For example, it can be done by separating the duplicate code to a new function or assigning `compositionProvider.Composition || compositionProvider.Composition$` to a temporary variable.

Pls add a `console.warn` to `saveLayout` saying that this method uses a deprecated way of storing custom compositions that will be removed few weeks after https://github.com/Starcounter/Blending/pull/395 is released in BlendingProvider.

**Comment by PatrykSzwer** [comment]
I was actually asking Omar how to do that DRY part best bc I wasn't sure how it should be done in JS, but you were faster with this `assigning compositionProvider.Composition || compositionProvider.Composition$` comment. Thanks!

Good idea with console warn. Added that. Let me know if this message sounds cool to you

**Review by warpech — APPROVED** [review]
Lgtm. @tomalec maybe you want to take a look at this PR, too?

File: starcounter-include.html [hunk 1]
```diff
@@ -462,6 +463,7 @@
          * @TODO: to be simplified after https://github.com/StarcounterApps/CompositionProvider/pull/13 is done
          */
         StarcounterIncludePrototype.saveLayout = function(compositionStr){
+			console.warn("This method of saving composition is deprecated and will be removed soon.");
```
Comment (alshakero) [inline starcounter-include.html]: I think this warning is useless without linking to or explaining an alternative. If there isn't one yet, I'd just remove it

**Review by alshakero — APPROVED** [review]
One minor comment

File: starcounter-include.html (hunk 1 above)
Comment (PatrykSzwer) [reply starcounter-include.html]: I updated the comment and @alshakero agreed with my correction. Merging.

**Pull request merged**